index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Final Trial</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+
+
+
+
+    <section id="welcome-section">
+     <a href="#projects"> <nav id="navbar">Go to Projects</nav></a>
+<h1 id="title">Hey were are Billal and Kevin</h1>
+ </section>
+    <section id="projects">
+      <div class="project-tile">
+  <a href="google.com" target="_blank"><p>Sample project</p></a> 
+      </div>
+     <a id="profile-link" href="https://github.com/bill7pearl/final-trial" target="_blank"> <p>My portfolio</p></a>
+    </section>
+
+
+
+    <div id="header">
+      <ul>
+        <li>Contact</li>
+        <li>Work</li>
+        <li>About</li>
+      </ul>
+    </div>
+
+    <div id="introduction">
+      <h1>We Are Billal and Kevin</h1>
+      <h2>Web developers</h2>
+    </div>
+
+    <div id="ourProjects">
+      <h1>These are our projects</h1>
+      <button>Show all</button>
+    </div>
+
+    <div id="contact">
+      <h1>Let's work together</h1>
+      <p>How do you take your coffee?</p>
+      <button>Facebook</button>
+      <button>Github</button>
+      <button>Twitter</button>
+      <button>Send a mail</button>
+      <button>Call me</button>
```

Comment:
Use unordered lists,<ul> instead of buttons to list the social pages and add a link in the list with an anchor tag to direct the user to your social media pages